content/docs/concepts/clusters.mdx

```diff
@@ -0,0 +1,103 @@
+---
+id: clusters
+title: Clusters
+description: This document describes Clusters in Pomerium Zero.
+keywords: [clusters, pomerium zero]
+sidebar_label: Clusters
+sidebar_position: 1
+---
+
+# Clusters in Pomerium Zero
+
+This document describes Clusters in Pomerium Zero.
+
+TODO: include a video that walks through Clusters
+
+## Clusters overview
+
+A **Cluster** represents a single Pomerium deployment managed by Pomerium Zero, our hosted control plane. Pomerium Zero enables you to configure cluster settings for your personal account or organization.
+
+Each cluster gets its own:
+
+- **Cluster identity**
+- **Cluster domain**
+- **Cluster settings**
+- **Routes and Policies**
+- **Wildcard certificates**
+
+### Cluster architecture
+
+TODO: It would be ideal to get an architecture diagram here.
+
+### Cluster identity
+
+When you create an account in Pomerium Zero, a new cluster with its own **Cluster Identity** is generated and assigned to your personal account. Pomerium Zero tethers this cluster to your account with the **Pomerium Zero Token** value provided to you during onboarding.
+
+This cluster identity token acts as a refresh token that you can use to generate a new cluster identity token in the Zero Console (see the [**Rotate cluster identity**](#rotate-cluster-identity) steps below).
+
+#### Rotate cluster identity
+
+A cluster identity token does not have a set expiration time. However, you may want to rotate the cluster identity in the Zero Console for security reasons. After you rotate the cluster identity, the previous token becomes obsolete.
+
+<details>
+  <summary>How to rotate cluster identity in the Zero Console</summary>
+  <div>
+    <ol>
+      <li>From the navigation bar dropdown, select **Manage Clusters**</li>
+      <li>Under **Actions**, select the **edit** icon</li>
+      <li>Select the **Domains** tab</li>
+      <li>In the **Cluster Identity** field, select **Rotate Identity**</li>
+      <li>Copy the token and replace the old one in your configuration file</li>
+    </ol>
+  </div>
+</details>
+
+### Cluster domain
+
+Pomerium Zero generates and assigns a unique **Cluster Domain** to your cluster. Your cluster domain contains your randomly generated **Cluster Name** and the `pomerium.app` subdomain.
+
+It follows this format: `<CLUSTER-NAME-XXXX>.pomerium.app`
+
+For example: `voracious-ape-1578.pomerium.app`
+
+#### Change cluster Name
+
+If you don't like your randomly generated cluster name, you can change it manually in your cluster settings.
```

Comment:
To me this almost reads like you can change the starter domain by changing the cluster name, which I don't think is the case.

I wonder if this would be a good place to instead link to a guide for adding a custom domain.